### PR TITLE
fix prometheus part

### PR DESCRIPTION
**Pull request by richardwxn**
the namespace and svc name need to update after using istioctl to install

### CODE REVIEW

File: perf/benchmark/run_benchmark_job.sh [hunk 1]
```diff
@@ -50,7 +51,7 @@ function setup_metrics() {
     export FORTIO_CLIENT_URL=http://localhost:8080
   fi
   export PROMETHEUS_URL=http://localhost:9090
-  kubectl -n istio-prometheus port-forward svc/istio-prometheus 9090:9090 &>/dev/null &
+  kubectl -n "${PROMETHEUS_NAMESPACE}" port-forward svc/prometheus 9090:9090 &>/dev/null &
```
Comment (carolynhu) [inline perf/benchmark/run_benchmark_job.sh]: So, when using istioctl, Prometheus is install directly under istio-system namespace?

File: perf/benchmark/run_benchmark_job.sh [hunk 2]
```diff
@@ -40,6 +40,7 @@ export PILOT_CLUSTER="${PILOT_CLUSTER:-}"
 export USE_MASON_RESOURCE="${USE_MASON_RESOURCE:-True}"
 export CLEAN_CLUSTERS="${CLEAN_CLUSTERS:-True}"
 export NAMESPACE=${NAMESPACE:-'twopods-istio'}
+export PROMETHEUS_NAMESPACE=${NAMESPACE:-'istio-system'}
```
Comment (mandarjog) [inline perf/benchmark/run_benchmark_job.sh]: This means usually Prometheus is in twopods namespace ?

Comment (richardwxn) [reply perf/benchmark/run_benchmark_job.sh]: good catch, fixed it

File: perf/benchmark/run_benchmark_job.sh (hunk 1 above)
Comment (richardwxn) [reply perf/benchmark/run_benchmark_job.sh]: yes

**Pull request merged**